src/include/sof/alh.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#ifndef __INCLUDE_ALH__
```

Comment:
Should go to sof/drivers/alh.h.
And also look at all other guards: `__SOF_DRIVERS_ALH_H__`